capabilities/controller/main.go

```diff
@@ -17,8 +17,8 @@ import (
 	// +kubebuilder:scaffold:imports
 
 	runv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/run/v1alpha1"
-	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/buildinfo"
-	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/sdk/capabilities/controllers"
+	capabilitiesControllers "github.com/vmware-tanzu/tanzu-framework/capabilities/controller/pkg/capabilities"
```

Comment:
usually, import aliases are not camel case. Also, this seems really long for an alias.

Some suggestions (and other ideas welcome):
- capctrl
- **capctl**
- capctlr
- cbctrl
- cbctlr
- cctrl
- cctlr

I put my favorite in bold.

Reply:
updated